src/test/rpc/Book_test.cpp

```diff
@@ -800,6 +800,460 @@ class Book_test : public beast::unit_test::suite
             books)[jss::status] == "success");
     }
 
+    void testTrackOffers(void)
+    {
```

Comment:
I would, yes. `testcase` has less value now that `BEAST_EXPECT` outputs line and file info when a test fails, but it's still useful.